static/app/components/assigneeSelector.tsx

```diff
@@ -296,22 +296,46 @@ class AssigneeSelector extends React.Component<Props, State> {
   renderNewDropdownItems(): ItemsBeforeFilter {
     const teams = this.renderNewTeamNodes();
     const members = this.renderNewMemberNodes();
+    const suggestedAssignees = new Set(this.renderSuggestedAssigneeNodes() ?? []);
+    const assigneesSearchKey: string[] = [];
```

Comment:
Since we check for inclusion on `assigneesSearchKey` perhaps we should make that a Set. Additionally, it doesn't seem like we ever check for inclusion on the `suggestedAssignees` set, so that could probably just be an array